modules/nf-core/trimmomatic/main.nf

```diff
@@ -13,7 +13,8 @@ process TRIMMOMATIC {
     output:
     tuple val(meta), path("*.paired.trim*.fastq.gz")   , emit: trimmed_reads
     tuple val(meta), path("*.unpaired.trim_*.fastq.gz"), optional:true, emit: unpaired_reads
-    tuple val(meta), path("*.log")                     , emit: log
+    tuple val(meta), path("*_trim.log")                , emit: trim_log
+    tuple val(meta), path("*_out.log")                 , emit: log
```

Comment:
This would mean that upon updating the module the current `log` channel now has a completely different log in it? Think it needs a different name to `log`

Reply:
Good call. Happy to swap the two names for backwards compatibility.

Reply:
Could be something like `out_log` and `trim_log` if that is sensible. 
I've never used trimmomatic, can you give an example of the relevant bits of the two logs?

Reply:
Sorry I intended to post this comment earlier with that commit. 

Here is the trim log generated using Trimmomatic's `-trimlog` command line argument:
```bash
VH00278:1:AAAN5CFM5:1:1101:61025:1114 1:N:0:ATTCGCACTA+TCGGTATGGT 113 0 113 37
VH00278:1:AAAN5CFM5:1:1101:61025:1114 2:N:0:ATTCGCACTA+TCGGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:64055:1114 1:N:0:ATTCGCACTA+TCTGTATGGT 115 0 115 36
VH00278:1:AAAN5CFM5:1:1101:64055:1114 2:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:63373:1151 1:N:0:ATTCGCACTA+TCTGTATGGT 151 0 151 0
VH00278:1:AAAN5CFM5:1:1101:63373:1151 2:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:55742:1170 1:N:0:ATTCGCACTA+TCTGTATGGT 149 0 149 2
VH00278:1:AAAN5CFM5:1:1101:55742:1170 2:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:58962:1170 1:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:58962:1170 2:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:59302:1170 1:N:0:ATTCGCACTA+TCTGTATGGT 46 0 46 104
VH00278:1:AAAN5CFM5:1:1101:59302:1170 2:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:63998:1170 1:N:0:ATTCGCACTA+TCTGTATGGT 27 0 27 124
VH00278:1:AAAN5CFM5:1:1101:63998:1170 2:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:54095:1189 1:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
VH00278:1:AAAN5CFM5:1:1101:54095:1189 2:N:0:ATTCGCACTA+TCTGTATGGT 0 0 0 0
```

And here is the stderr log:

```bash
TrimmomaticPE: Started with arguments:
 -threads 6 -trimlog CFIABU2016-0360-1.log -summary CFIABU2016-0360-1.summary CFIABU2016-0360-1_S809_L001_R1_001.fastq.gz CFIABU2016-0360-1_S809_L001_R2_001.fastq.gz CFIABU2016-0360-1.paired.trim_1.fastq.gz CFIABU2016-0360-1.unpaired.trim_1.fastq.gz CFIABU2016-0360-1.paired.trim_2.fastq.gz CFIABU2016-0360-1.unpaired.trim_2.fastq.gz SLIDINGWINDOW:4:25 MINLEN:25 TRAILING:25
Quality encoding detected as phred33
Input Read Pairs: 3476017 Both Surviving: 3116831 (89.67%) Forward Only Surviving: 162792 (4.68%) Reverse Only Surviving: 73178 (2.11%) Dropped: 123216 (3.54%)
TrimmomaticPE: Completed successfully
``` 

So quite vastly different. Interestingly, MultiQC only needs the high level summary info found in the stderr log, not the extremely detailed trim log (I double-checked this earlier this morning).